src/main/java/com/google/sps/servlets/DestinationDataServlet.java

```diff
@@ -44,10 +46,11 @@
   private static final String HINT3_PARAMETER = "hint3";
   private static final String OBSCURITY_PARAMETER = "obscurity";
   private static final String TAG_PARAMETER = "tag";
-  private static final String REDIRECT_URL = "/destination-data";
+  private static final String HOME_URL = "/index.html";
 
-  // Temporarily stores the destination created by the user
-  public List<Destination> destinations = new ArrayList<>();
+  private static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();
+
+  private static final Gson GSON = new Gson();
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I think that this method should have its own Javadoc and that the class-level Javadoc should be updated now that the behavior is more fleshed out. I think that the class-level should be more general, like `Stores a destination in Datastore`, and the method-level should give a bit of detail in terms of its input and output, including: takes various parameters from the request, combines them to create a json string to represent a destination, stores the string in the datastore, and sends a redirect to HOME_URL.